bucket/pov-ray.json

```diff
@@ -0,0 +1,65 @@
+{
+    "homepage": "https://www.povray.org/",
+    "version": "3.7.0",
+    "license": "AGPL-3.0-or-later",
+    "description": "A high-quality tool for creating stunning three-dimensional graphics.",
+    "url": [
+        "https://www.povray.org/ftp/pub/povray/Official/povwin-3.7-agpl3-setup.exe",
+        "https://www.povray.org/download/povwin-3.7-editor.exe"
+    ],
+    "hash": [
+        "2b1331641b6f96113c2dba951be80d99eb548639480399cabb5e3a60dce5bdc8",
+        "9265f0d3337f956ae42c1bc19b475182c3c5e543e96df2287450afb9738b6030"
+    ],
+    "installer": {
+        "script": [
+            "if (!(Test-Path \"$persist_dir\\userdocs\")) {",
+            "   Expand-7zipArchive -Path \"$dir\\*-setup.exe\" -DestinationPath \"$dir\\userdocs\" -ExtractDir '$_15_'",
+            "}",
+            "Expand-7zipArchive -Path \"$dir\\*-setup.exe\" -ExtractDir '$_14_' -Removal",
+            "Expand-7zipArchive -Path \"$dir\\*-editor.exe\" -DestinationPath \"$dir\\bin\" -ExtractDir '$_12_' -Removal",
+            "Remove-Item \"$dir\\`$*\" -Recurse -Force"
+        ]
+    },
+    "architecture": {
+        "64bit": {
+            "bin": [
+                "bin\\pvengine64.exe",
+                [
+                    "bin\\pvengine64.exe",
+                    "pvengine"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "bin\\pvengine64.exe",
+                    "POV-Ray"
+                ]
+            ]
+        },
+        "32bit": {
+            "bin": [
+                "bin\\pvengine32-sse2.exe",
+                [
+                    "bin\\pvengine32-sse2.exe",
+                    "pvengine"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "bin\\pvengine32-sse2.exe",
+                    "POV-Ray"
+                ]
+            ]
+        }
+    },
+    "persist": "userdocs",
+    "post_install": "pvengine /QINSTALL \"$dir\" \"$dir\\userdocs\"",
```

Comment:
What is this for?

Reply:
The installer contains two parts: programs and some user document files (in `$_14_` and `$_15_`). Default location of user document files (`$_15_`) is `Documents\POV-Ray\vX.y`, and `pvengine /INSTALL [home dir] [user dir]` (`/QINSTALL` means quiet) change these locations (Ref: http://wiki.povray.org/content/Documentation:Windows_Section_3#Non-Standard_Installations)

Reply:
The seperate `editor` download url 'cos it has different license that cannot packed into one package...